worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -741,6 +748,31 @@ private[celeborn] class Worker(
       })
     }
 
+  private def removeAppResourceConsumption(applicationIds: JHashSet[String]): Unit = {
+    applicationIds.asScala.foreach { applicationId =>
+      removeResourceConsumptionGauge(
+        ResourceConsumptionSource.DISK_FILE_COUNT,
+        applicationId)
+      removeResourceConsumptionGauge(
+        ResourceConsumptionSource.DISK_BYTES_WRITTEN,
+        applicationId)
+      removeResourceConsumptionGauge(
+        ResourceConsumptionSource.HDFS_FILE_COUNT,
+        applicationId)
+      removeResourceConsumptionGauge(
+        ResourceConsumptionSource.HDFS_BYTES_WRITTEN,
+        applicationId)
+    }
+  }
+
+  private def removeResourceConsumptionGauge(
+      resourceConsumptionName: String,
+      applicationId: String): Unit = {
+    resourceConsumptionSource.removeGauge(
+      resourceConsumptionName,
+      Map(resourceConsumptionSource.applicationLabel -> applicationId))
```

Comment:
Since CELEBORN-1485, removeGauge no longer supports subset labels, so this code will not remove gauge.

Reply:
You need to get the user of this application, and removeGauge like that:
```
resourceConsumptionSource.removeGauge(
      resourceConsumptionName,
      userIdentifier.toMap + (resourceConsumptionSource.applicationLabel -> applicationId))
```

Reply:
@leixm, sorry for forgetting the changes of `removeGauge`. I have updated `removeAppResourceConsumption` to support removing resource consumption metrics.

Reply:
Can we introduce a mapping relationship between applicationId and user? This can reduce cpu time.